core-addon/IonCore.js

```diff
@@ -242,4 +254,24 @@ module.exports = class IonCore {
 
     return await this._sendEmptyPing("deletion-request", studyAddonId);
   }
+
+  /**
+   * Fetch the available studies.
+   *
+   * This loads the studies from the Firefox Remote Settings service.
+   *
+   * @returns {Promise(Array<Object>)} resolved with an array of studies
+   *          objects, or an empty array on failures.
+   */
+  async _fetchAvailableStudies() {
+    try {
+      const request = await fetch(
+        "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/pioneer-study-addons-v1/records"
```

Comment:
Have you thought about making this URL configurable? It's specified in the manifest (so we can do `fetch` without CORS issues) and also here so I guess it'd be done at build time? I mention it because I am pretty sure we'll want to point at the dev and stage remotesettings servers for QA purposes.

Reply:
Makes sense. I filed #44 for this.